app/javascript/packs/openid-connect-redirect.js

```diff
@@ -0,0 +1,4 @@
+document.addEventListener('DOMContentLoaded', function () {
+  document.body.className += ' usa-sr-only';
+  document.getElementById('openid-connect-redirect').click();
+});
```

Comment:
Few points of feedback, acknowledging that this is likely inheriting mostly from `saml-post.js`:

- I'd recommend against adding any new `.js` files, I think we could make this `.ts` without too much trouble
- We can use `classList` over `className` manipulation to avoid dealing with space-separated tokens
- I think we should move the `render_javascript_pack_once_tags` to the bottom of the `<body>` tag (best practice), at which point we could avoid `DOMContentLoaded` event handler
- I like using the existing link as a reference for the navigation, but I'm not as sure about relying on the click mechanic vs. a more explicit call to navigation APIs like `window.location =` (or the `url` package's `forceNavigate`)

```suggestion
import { forceNavigate } from '@18f/identity-url';

document.body.classList.add('usa-sr-only');

const link = document.getElementById('openid-connect-redirect') as HTMLLinkElement;
forceRedirect(link.href);
```